src/index.html

```diff
@@ -1,14 +1,57 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="menu">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="index.html">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="index.html">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="index.html">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a 
+            class="nav_link" 
+            data-qa="hover" 
+            href="index.html"
+            >Laptops & computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="index.html">Gadgets</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="index.html">Tablets</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="index.html">Photo</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="index.html">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <a class="logo_link" href="index.html">
+      <img src="./images/logo.png" alt="logo">
+    </a>
```

Comment:
why its on the end if its placed on the beginning of the header + move it to the header block?